src/include/sof/audio/component.h

```diff
@@ -512,6 +512,7 @@ static inline void sys_comp_src_init(void) {}
 void sys_comp_tone_init(void);
 void sys_comp_eq_iir_init(void);
 void sys_comp_eq_fir_init(void);
+void sys_comp_keyword_init(void);
```

Comment:
Please make it configurable with Kconfig
Example: https://github.com/thesofproject/sof/pull/955/commits

Reply:
Very well, will do